hack/verify-updates.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+make update
+diff="$(git diff)"
+
+if [ ! -z "$diff" ]; then
```

Comment:
Please refer to https://github.com/grawlinson/version-embedding/blob/806e1c43d5fde62f67ce14e5ddfce2fca428c329/Makefile#L16
I think `git status` is widely used in this scenario.

Reply:
Good idea. Updated.